web_hierarchy/static/src/js/view/web_hierarchy_view.js

```diff
@@ -0,0 +1,95 @@
+ /*
+ * Copyright 2019 Kevin Kamau <kevkamaa96@gmail.com>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+ */
+
+odoo.define('web.HierarchyView', function (require) {
+    'use strict';
+
+    var core = require("web.core");
+    var View = require('web.View');
+    var _lt = core._lt;
+    var qweb = core.qweb;
+
+    var HierarchyView = View.extend({
+        template: 'HierarchyView',
+        icon: "fa-sitemap",
+        searchable: false,
+        mobile_friendly: true,
+        require_fields: true,
+        display_name: _lt('Hierarchy'),
+        render_chart: function (size) {
+            var self = this;
+            var size = (!size) ? "medium": size;  // default value
+            var parent_name = this.options.action.display_name;
+            google.charts.load('current', {packages: ["orgchart"]});
+            google.charts.setOnLoadCallback(drawChart);
```

Comment:
Are these charts included in Odoo by default?

Reply:
no they use google chart specifically the org chart.